agent/tcs/client/client.go

```diff
@@ -153,13 +153,6 @@ func (cs *clientServer) publishMetrics() {
 		return
 	}
 
-	// Publish metrics immediately after we connect and wait for ticks. This makes
-	// sure that there is no data loss when a scheduled metrics publishing fails
-	// due to a connection reset.
-	err := cs.publishMetricsOnce()
```

Comment:
This comment and function call no longer applies, this function does not get recalled on connection reset. Trying to publish metrics on startup won't ever work because there are no metrics in the queue yet.